android-exoplayer/src/main/java/com/brentvatne/exoplayer/ExoPlayerView.java

```diff
@@ -261,7 +266,20 @@ private final class ComponentListener implements VideoListener,
 
         @Override
         public void onCues(List<Cue> cues) {
-            subtitleLayout.onCues(cues);
+            // figma design baseline is 1080p
+            int screenHeight = 1080;
+            int lineHeightPixels = 42;
+            float pixelRatio = 77 / (float) screenHeight;
```

Comment:
why do we need to cast to float here?

Reply:
We need to cast to float because we are doing a division operation between two integer values. In Java, if you divide an integer by an integer, the result will also be an integer. This can lead to problems with precision

Reply:
sorry i meant this part `(float) screenHeight`

Reply:
interesting